xmlserializer/XmlDocSource.h

```diff
@@ -57,20 +57,18 @@ class CXmlDocSource : private utility::NonCopyable
       * @param[in] pRootNode a pointer to the root element of the document.
       * @param[in] bValidateWithSchema a boolean that toggles schema validation
       */
-    CXmlDocSource(_xmlDoc* pDoc, bool bValidateWithSchema = false, _xmlNode* pRootNode = NULL);
+    CXmlDocSource(_xmlDoc* pDoc, bool bValidateWithSchema, _xmlNode* pRootNode);
```

Comment:
This constructor seems unused. If so, you may remove it altogether (in a separate patch).